lbz/events/broker.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Callable, Dict, List
+
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass()
+class Event:
+    type: str
+    data: dict
+
+
+class EventBroker:
+    def __init__(self, mapper: Dict[str, List[Callable]], raw_event: dict) -> None:
```

Comment:
I liked the implementation in which `raw_event` was passed directly to the `handle()` method.

Thanks to that we will be able to handle multiple events using one object of this class - more natural 😉

Reply:
Lambda does not allow for such case ATM.
1 event = 1 invocation

Reply:
and above give you access to the even at all times.

Reply:
I mentioned it from the code/Python perspective, not Lambda 😉 

And yes, there are benefits of both solutions, so definitely, I won't force changing that 👍